### DISCUSSION TITLE
the url from GENERAL admin page is not working

### DISCUSSION THREAD

**Discussion by darashayda** [requarks/wiki#5970] [Help / Questions] [answered]
https://wiki.ccnstudios.com  

This is supposed to be our wiki page for documentation at address

http://137.184.127.233//freeform/intro  

However https://wiki.ccnstudios.com  issues server not found message! 

We are using Digital Ocean for hosting... 

Dara

**Comment by NGPixel** [comment, marked as answer]
Have you created an A Record on your domain registrar to point to that IP?

**Reply by darashayda** [reply]
Ok we CAN do this the way Wiki.js has been setup

https://wiki.ccnstudios.com/freeform/intro

But I need to do the top directory, below attached image namely the root that contains the CloudFunc and freeform. 

I am sure some step in the beginning was missed or not setup with the right names by me, but I need to remedy it. 

I prefer the top directory to be

freeform 

underneath 

intro
Cloudfunc
Geometry 

...


and I like the address to be 

https://wiki.ccnstudios.com/freeform

Do I make sense?


<img width="957" alt="Screen Shot 2022-12-20 at 10 10 20 PM" src="https://user-images.githubusercontent.com/5667455/208776541-abac5495-e1ea-4323-afb7-986d3c7f9ec3.png">

**Reply by darashayda** [reply]
Sorry I nuked the pages to start fresh, I do not have time to wait days for a reply. 

D

**Reply by darashayda** [reply]
Did not mean to be critical, we need to move faster

**Comment by darashayda** [comment]
No I did not! I know what you are referring to so I assume I do that first. 

Let me take  care of that and be back. 

Thanx

Dara

**Comment by darashayda** [comment]
@NGPixel 

Thank you for the reminder about the domain upgrades, we did and all is working now .

http://wiki.ccnstudios.com/home/freeform/intro  

We will bog you with a few little details in coming 2 weeks and probably that is all.

Pleased with the product, I wish the suppoert service to be a bit more responsive. 

D

**Reply by NGPixel** [reply]
This is an open source project run by **volunteers** on their **free time**. There's no "support service" nor is there any expectation to reply to everyone in a set time frame. So either adjust your expectations on the matter or use a commercial product instead.

**Reply by darashayda** [reply]
In that case , thank you for your generous contribution much obliged. 

No not interested in commercial products, we use Wiki.Js for next 2 months, unless some disaster, we will stay for years to come. 

Again thank you, and it is great product. 

Dara
